config/entries.go

```diff
@@ -41,6 +41,7 @@ var ClientId = asString("daemon.auth.clientId", "")
 var ClientSecret = asString("daemon.auth.clientSecret", "")
 var CacheFolder = asDataFolder("daemon.data.cache", "cache")
 var ServersFolder = asDataFolder("daemon.data.servers", "servers")
+var BackupsFolder = asDataFolder("daemon.data.backups", "backups")
```

Comment:
Thinking to the future, I think we probably should have this be an object instead... because eventually it'd be "add s3" and such.

Can we have this be daemon.data.backups.folder to prep for that.